docs/components/modeler/web-modeler/git-sync.md

```diff
@@ -65,15 +80,67 @@ When successful, your project will display a new **Sync with GitHub** button.
 
 ![The Sync with GitHub within Web Modeler](./img/git-sync.png)
 
-## Sync with GitHub
+</TabItem>
+<TabItem value='gitlab'>
+
+<h3> Create a new project access token </h3>
+
+Web Modeler requires a project access token to sync changes with your GitLab repository.
+
+Follow the [GitLab documentation](https://docs.gitlab.com/ee/user/project/settings/project_access_tokens.html#create-a-project-access-token) to generate a new project access token for your repository with the following configuration:
+
+- Enable the following [**scopes**](https://docs.gitlab.com/ee/user/project/settings/project_access_tokens.html#scopes-for-a-project-access-token):
+  - `api`
+  - `read_repository`
+  - `write_repository`
+  - `read_registry`
+  - `write_registry`
```

Comment:
🔧 Some of these permission are not needed (in my opinion). `api`, `read_api`, `read_repository`, `write_repository` should be enough but correct my if I missed something.

Reply:
This is what I used for testing. When the implementation is done I will revisit the page and update if we need more permissions than I assumed.

Reply:
okay, they are udpated now.